tests/utils/autoTester.js

```diff
@@ -413,15 +413,13 @@ module.exports.getOriginFrame = function () {
     return originFrame;
 }
 
-// Utilities to provide GitHub branch and repository (i.e. GitHub path to "tests")
+// Utilities to provide GitHub information
 var _user;
-var _repository;
 var _branch;
 var _repositoryPath;
 
-module.exports.setRepositoryInfo = function (user, repository, branch, repositoryPath) {
+module.exports.setRepositoryInfo = function (user, repositoryPath, branch) {
```

Comment:
just for consistency can you either go for:
repositoryPath, user, branch
or
user, branch, repositoryPath

I don't quite understand why you need the user or the branch in autoTester really but your call